editor/scene_tree_dock.cpp

```diff
@@ -77,7 +77,7 @@ void SceneTreeDock::_quick_open() {
 
 void SceneTreeDock::_inspect_hovered_node() {
 	select_node_hovered_at_end_of_drag = true;
-	if (tree_item_inspected != nullptr) {
+	if (tree_item_inspected) {
 		tree_item_inspected->clear_custom_color(0);
```

Comment:
I was thinking that it did not matters there because it is only to clear the custom color if you inspect another tree item in the same drag operation and thus I was thinking that you can't really invalidate it while doing the drag, but I guess  you could still use arrow keys and press delete so yes I guess it should be set to `nullptr` as well to be safe.

Reply:
I will move this a bit lower in the code instead, that way it will be clearer when it get set/reset